checks.d/kafka_consumer.py

```diff
@@ -1,20 +1,29 @@
 # stdlib
 from collections import defaultdict
-import random
 
-# project
-from checks import AgentCheck
-
-# 3rd party
+# 3p
 from kafka.client import KafkaClient
 from kafka.common import OffsetRequest
 from kazoo.client import KazooClient
 from kazoo.exceptions import NoNodeError
 
+# project
+from checks import AgentCheck
+
+DEFAULT_KAFKA_TIMEOUT = 5
+DEFAULT_ZK_TIMEOUT = 5
+
+
 class KafkaCheck(AgentCheck):
 
     SOURCE_TYPE_NAME = 'kafka'
 
+    def __init__(self, name, init_config, agentConfig, instances=None):
+        self.zk_timeout = int(
+            init_config.get('zk_timeout', DEFAULT_ZK_TIMEOUT))
+        self.kafka_timeout = int(
+            init_config.get('kafka_timeout', DEFAULT_KAFKA_TIMEOUT))
+
```

Comment:
You forgot the `AgentCheck.__init__(self, name, init_config, agentConfig, instances=instances)`.